package.json

```diff
@@ -50,7 +50,8 @@
     "tailwindcss": "^3.2.7",
     "uuid": "^9.0.0",
     "wasm-loader": "^1.3.0",
-    "web-vitals": "^2.1.4"
+    "web-vitals": "^2.1.4",
+    "gluon-sdk": "file:../gluon-sdk"
```

Comment:
Does this mean that the SDK needs to be manually included in the folder structure of the project?  Can't we publish it and then import it like a normal javascript library?